protoc_plugin/pubspec.yaml

```diff
@@ -10,7 +10,7 @@ environment:
 dependencies:
   fixnum: ^0.10.5
```

Comment:
Any reason not to update this to `^10.9` as well?

Reply:
Also done.